src/api.js

```diff
@@ -18,26 +18,42 @@ module.exports = class Hub {
 			throw new Error('Missing property tenant')
 		}
 
-		const uri = new URL(options.path, `https://${this.options.tenant}`).href
+		const {path, qs, json, headers = {}, ...opts} = options
 
-		options.uri = uri
+		// URL Path
+		const url = new URL(path, `https://${this.options.tenant}`)
 
-		if (!('json' in options)) {
-			options.json = true
+		// How to handle JSON
+		// Explitly request JSON response
+		const convertToJSON = json ?? true
+		if (convertToJSON) {
```

Comment:
`if (json) {`

Reply:
I think you haven't noted the nullish coalescing `?? true`